main.py

```diff
@@ -34,8 +40,9 @@ def process_month(self, year, month):
         raw_data = self.file_reader.read_weather_file(file)
         weather_data = [WeatherData.from_csv_row(row) for row in raw_data]
 
-        avg_high_temp, avg_low_temp, avg_mean_humidity = WeatherStats.calculate_averages(
-            weather_data)
+        avg_high_temp, avg_low_temp, avg_mean_humidity = (
+            WeatherStats.calculate_averages(weather_data)
+        )
```

Comment:
Keep the same format. Like in process year you distribute variables into multi lines to ensure the indentation and here you have choosen a different method. Try to be consistent with whatever coding practice you choose.